processor/src/fast/tests/snapshots/miden_processor__fast__tests__all_ops__fast::tests::all_ops::test_basic_block::stack_inputs_05_vec__0_u32_into___1_u32_into___2_u32_into___3_u32_into___::operations_66_vec__Operation__FriE2F4_.snap

```diff
@@ -1,26 +1,8 @@
 ---
 source: processor/src/fast/tests/all_ops.rs
+assertion_line: 133
 expression: fast_stack_outputs
 ---
-Ok(
-    StackOutputs {
-        elements: [
-            9223372034707292161,
-            0,
-            9223372034707292162,
-            1,
-            0,
-            0,
-            0,
-            1,
-            0,
-            1,
-            8,
-            0,
-            0,
-            1,
-            9223372034707292161,
-            0,
-        ],
-    },
+Err(
+    InvalidFriDomainGenerator,
```

Comment:
Here, and in other places, it seems like there was some error - but all the tests are passing. How should I interpret this?